util/anonymize/anonymize.go

```diff
@@ -0,0 +1,212 @@
+package anonymize
```

Comment:
Why did you place the anonymize into the root folder/util? In the client folder fits better.